src/services/anchor/index.ts

```diff
@@ -77,9 +79,20 @@ export const sep10 = async (
   const NETWORK_PASSPHRASE = Networks.PUBLIC;
   const ephemeralKeys = Keypair.fromSecret(stellarEphemeralSecret);
   const accountId = ephemeralKeys.publicKey();
-  const urlParams = new URLSearchParams({
-    account: accountId,
-  });
+
+  let urlParams;
+  if (outputToken === 'ars') {
```

Comment:
I think refactoring this to a function and then `const urlParams = theFunction()` would improve readability and reusability

Reply:
Alright! I will add it also including this [comment](https://github.com/pendulum-chain/vortex/pull/201#discussion_r1829797828).